definitions/subworkflows/varscan.cwl

```diff
@@ -82,14 +82,14 @@ steps:
         out:
             [snvs, indels]
     process_somatic_snvs:
-        run: ../definitions/tools/varscan_processsomatic.cwl
```

Comment:
I wish we had renamed this workflow to `varscan_process_somatic.cwl`. I must've missed that in the original tools PR. Now might be a good time to do that?

Reply:
I'll fix that now